packages/arcgis-rest-portal/src/items/get.ts

```diff
@@ -148,13 +148,16 @@ export function getItemResources(
 ): Promise<any> {
   const url = `${getPortalUrl(requestOptions)}/content/items/${id}/resources`;
 
-  // mix in user supplied params
-  requestOptions.params = {
-    ...requestOptions.params,
-    num: 1000
+  // Mix in num:1000 with any user supplied params
+  // Key thing - we don't want to mutate the passed in requestOptions
+  // as that may be used in other (subsequent) calls in the course
+  // of a long promise chains
+  const options: IRequestOptions = {
+    ...requestOptions
   };
+  options.params = { ...options.params, ...{ num: 1000 } };
```

Comment:
I think you could skip one destructuring assignment and allow folks to set their own `num` if you did something like this instead.
```suggestion
  options.params = { num: 1000, ...options.params };
```

Reply:
good call... I was just bustin' out the 🔨 having spent an hour+ pulling my hair out tracking down what was janking the requestOptions in the middle of a long promise chain... causing fetchMock to not match a subsequent search api call... but - commit inbound

Reply:
the inverted order preceded your hammering.

nice sleuthing! 🚧🕵👷🚧

Reply:
updated & thanks! hope all is well @jgravois :)